types/mjml-react/index.d.ts

```diff
@@ -90,6 +90,11 @@ export class MjmlAttributes extends React.Component<{ children?: React.ReactNode
 export class MjmlAll extends React.Component<{ [key: string]: any; children?: React.ReactNode | undefined }> { }
 export class MjmlClass extends React.Component<{ [key: string]: any; children?: React.ReactNode | undefined; name: string }> { }
 
+// mj-html-attributes
+export class MjmlHtmlAttributes extends React.Component<{ children?: React.ReactNode }> { }
+export class MjmlSelector extends React.Component<{ children?: React.ReactNode | undefined; path: string }> { }
+export class MjmlHtmlAttribute extends React.Component<{ children?: React.ReactNode | undefined; name: string }> { }
```

Comment:
`MjmlHtmlAttributes`, `MjmlSelector` and `MjmlHtmlAttribute` components were introduced in https://github.com/wix-incubator/mjml-react/pull/72, inline with https://documentation.mjml.io/#mj-html-attributes (introduced in mjml v4.7.0)

Reply:
Looks good and makes sense. This seems like the core of the PR